lib/src/ledger/ledger_ble_manager.dart

```diff
@@ -56,13 +54,6 @@ class LedgerBleConnectionManager extends ConnectionManager {
       return;
     }
 
-    try {
-      await disconnect(device.id);
-    } catch (e) {
-      // ignore;
-      // may throw if device is not connected yet but that's fine
-    }
```

Comment:
Was this removed for any particular reason/issue? From my testing, connecting to a device that's not been (properly?) disconnected / disposed will cause the connection to hang. So doing a disconnect here made it work more reliable.

Reply:
It always throws the following exception which can't be caught at this time due to it being thrown during the Queue execution of UniversalBle.

`PlatformException(IllegalArgument, Unknown deviceId:
I/flutter (13066): DE:F1:9B:AA:4E:24, null, null)`